### PR TITLE
update geoplot.py to handle 1d or 2d q array, and also eta = q[-1,...]

### PR DESCRIPTION
**Pull request by rjleveque**
Use ellipses q[0,...] for depth, which is q[0,:] in 1d or q[0,:,:] in 2d. 

Use eta = q[-1,...] to work in 1d or 2d, and also with 2d bouss where q will have additional components for hu and hv correction terms.  Assumes that eta is always the final component of the q array read in from geoclaw output.

### CODE REVIEW

File: src/python/visclaw/geoplot.py [hunk 1]
```diff
@@ -2,15 +2,13 @@
...
-from __future__ import absolute_import
-from __future__ import print_function
 import numpy
 import warnings
 
 from clawpack.visclaw import colormaps
 from matplotlib.colors import Normalize
 from clawpack.geoclaw import topotools
-from six.moves import range, zip
+# from six.moves import range, zip
```
Comment (mandli) [inline src/python/visclaw/geoplot.py:11]: Might as well just remove this.

**Review by mandli — APPROVED** [review]
Looks good to me other than the minor comment.  Do you want to also include an example in this PR or just wait?

File: src/python/visclaw/geoplot.py (hunk 1 above)
Comment (rjleveque) [reply src/python/visclaw/geoplot.py]: done.

**Comment by rjleveque** [comment]
@mandli: Not sure what sort of example you mean since we don't have any examples in visclaw repo now.  For old examples in geoclaw it should work the same as before, I hope.

**Comment by mandli** [comment]
Sorry, was looking at too many PRs.

**Pull request merged**